src/pages/_app.js

```diff
@@ -3,14 +3,19 @@ import '../styles/global.scss';
 import { I18nextProvider } from 'react-i18next';
 import i18n from '../i18n';
 import { ThemeProvider } from '../components/publiq-ui/ThemeProvider';
+import { CookiesProvider } from 'react-cookie';
 
 // eslint-disable-next-line react/prop-types
-const App = ({ Component, pageProps }) => (
-  <I18nextProvider i18n={i18n}>
-    <ThemeProvider>
-      <Component {...pageProps} />
-    </ThemeProvider>
-  </I18nextProvider>
-);
+const App = ({ Component, pageProps }) => {
+  return (
+    <I18nextProvider i18n={i18n}>
+      <ThemeProvider>
+        <CookiesProvider>
+          <Component {...pageProps} />
+        </CookiesProvider>
```

Comment:
I guess choosing the cookie solution depends a lot of whether we need this Provider here (mainly for SSR reasons). If we want to go for a 100% SPA approach, we need to configure Next a bit more https://colinhacks.com/essays/building-a-spa-with-nextjs